lib/registrant.rb

```diff
@@ -0,0 +1,56 @@
+class Registrant
+
+    attr_reader :name, 
+                :age, 
+                :permit, 
```

Comment:
Careful here. We do not want a method exposed called `.permit`, we have `.permit?`, so we do not want this attr_reader